tests/main_test.py

```diff
@@ -83,13 +83,39 @@ def test_user_info(self):
         assert status_code == 200
         assert content['data']['pay_with'] == data['pay_with']
 
+    def test_group_no_desc(self):
+        """
+        test
+        :return:
+        """
+
+        # create a group
+        data = {
+            'name': 'test group name',
+        }
+        content, status_code = self.do_post('/group/create', {'data': data})
+        self.group = content['data']
+        assert status_code == 200
+
+    def test_bad_group(self):
+        """
+        test
+        :return:
+        """
+
+        # create a group with no data
+        data = {
+            'bad': 'data'
+        }
+        content, status_code = self.do_post('/group/create', {'data': data})
+        assert status_code == 400
```

Comment:
probably also want to add an additional assert for the msg that gets returned apart from just the status code (you can add as many asserts per test as you want), in order to define exactly what should be where in the response based on what you did

Reply:
added assert for content msg string